Machine-Learning/README.md

```diff
@@ -0,0 +1,106 @@
+# Machine Learning
+
+## Introduction  
+
+- [What is Machine Learning? -IBM.com](https://www.ibm.com/topics/machine-learning)
+- [What is Machine Learning? -MIT](https://mitsloan.mit.edu/ideas-made-to-matter/machine-learning-explained)
+
+## Getting Started with Python
+
+- [The Python tutorial -python.org](https://docs.python.org/3/tutorial/)
+- [Python in One Short -FreeCodeCamp(Youtube)](https://youtu.be/eWRfhZUzrAch)
+- [Python for absolute Begineer  -Corey Schafer(Full playlist including frameworks  and libraries)](https://youtube.com/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU)
+- [Intermidiate python in 6 hours -FreeCodeCamp](https://youtu.be/HGOBQPFzWKo)
+- [Python Tutorial -Tutorials point](https://www.tutorialspoint.com/python/index.html)
+- [Data Structures and algorithms in python -FreeCodeCamp](https://youtu.be/pkYVOmU3MgA)
+-[Robust Python: Write Clean and Maintainable Code by patrick viafore-pdf](https://dokumen.pub/download/robust-python-write-clean-and-maintainable-code-1nbsped-1098100662-9781098100667-c-7156903.html)
```

Comment:
This is a pirated PDf of the book.

Reply:
Remove this.

Reply:
Please note that we want to include only free resources in all our roadmaps. Pirated content is strictly prohibited.

Reply:
I am sorry for adding the pirated copy of the book